invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/SendMailModal/SendMailButton.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
again, IMHO, it is not a generic button `SendMailButton`, unless you pass a title as a prop, but it is everything very specific to sending email reminder for overdue loans. I would go for more specific, or fully generic.

Reply:
Fully specific it is the modal will render also the button.